index_ghost_host.html

```diff
@@ -0,0 +1,373 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" type="text/css" media="screen" href="css/ghost_host.css" />
+  <title>GhostHost</title>
+</head>
+
+<body class="wrapper">
+  <header class="wrapper">
+    <nav class="flexRow wrapper headerNav">
+      <a class="logo" href="markup-1.html">
+        <img src="img/Logo.png" alt="Ghost Host">
+      </a>
+      <div class="headerNavigation flexRow">
+        <a class="headerNavLink" href="#">Home</a>
+        <a class="headerNavLink" href="#">Domain</a>
+        <a class="headerNavLink" href="#">Hosting</a>
+        <a class="headerNavLink" href="#">Clients</a>
+        <a class="headerNavLink" href="#">Blogs</a>
+        <a class="headerNavLink" href="#">Support</a>
+      </div>
+    </nav>
+    <div class="greetingHeader wrapper">
+      <h1>Hello!</h1>
+      <p>We love our work...</p>
+      <p>And We're taking it seriously</p>
+    </div>
+    <div class="flexRow headerBottom">
+      <div class="headerBottomHalf">
+        <input class="headerButton" type="button" name="Login" value="Login">
+      </div>
+      <div class="headerBottomHalf ">
+        <input class="headerButton" type="button" name="Register" value="Register">
+      </div>
+
+    </div>
+  </header>
+  <main>
+    <!--Checking the domain-->
+    <section class="domainChoice">
+      <div class="wrapper">
+        <p class="domainChoiceHead">Would you like to buy a new domain?</p>
+        <p class="domainChoiceBody">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+          incididunt ut labore</p>
+      </div>
+      <div class="flexRow domainChoiceRow wrapper">
+        <div class="urlFirst domainChoicePart">
```

Comment:
```suggestion
        <span class="urlFirst domainChoicePart">
```
Don't put text to div